File: CS/MasterDetailProject/Order.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MasterDetailProject {
    class Order {
        public Order(int id, string name, DateTime date) {
            this.Id = id;
            this.Name = name;
            this.Date = date;
        }

        private int _age;
        private int _id;
        public int Id {
            get {
                return _id;
            }
            set {
                _id = value;
            }
        }

        private string _name;
        public string Name {
            get {
                return _name;
            }
            set {
                _name = value;
            }
        }

        private DateTime _date;
        public DateTime Date {
            get {
                return _date;
            }
            set {
                _date = value;
            }
        }

    }
}

File: VB/MasterDetailProject/Order.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace MasterDetailProject
	Friend Class Order
		Public Sub New(ByVal id As Integer, ByVal name As String, ByVal [date] As DateTime)
			Me.Id = id
			Me.Name = name
			Me.Date = [date]
		End Sub

		Private _age As Integer
		Private _id As Integer
		Public Property Id() As Integer
			Get
				Return _id
			End Get
			Set(ByVal value As Integer)
				_id = value
			End Set
		End Property

		Private _name As String
		Public Property Name() As String
			Get
				Return _name
			End Get
			Set(ByVal value As String)
				_name = value
			End Set
		End Property

		Private _date As DateTime
		Public Property [Date]() As DateTime
			Get
				Return _date
			End Get
			Set(ByVal value As DateTime)
				_date = value
			End Set
		End Property

	End Class
End Namespace
